packages/manager/apps/web/client/app/email-domain/order/email-domain-order.html

```diff
@@ -1,122 +1,99 @@
+        <span data-translate="mxPlan_order_generated_success"></span>
+        <a data-ng-href="{{ $ctrl.orderTrackingLink }}" data-translate="mxPlan_order_generated_success_tracking"></a>
+    </oui-message>
+
+    <oui-stepper
+        data-current-index="$ctrl.currentIndex"
+        data-on-finish="$ctrl.validateCheckout(forms)">
+        <oui-step-form
+            data-header="{{:: 'mxPlan_order_domain' | translate }}"
+            data-valid="$ctrl.model.selectedDomain"
+            data-loading="$ctrl.loading.domain"
+            data-editable="!$ctrl.loading.checkout">
+            <oui-select
+                class="oui-select_m"
+                data-items="$ctrl.domains"
+                data-model="$ctrl.model.selectedDomain"
+                data-on-change="$ctrl.onDomainChange()"
+                data-placeholder="{{:: 'mxPlan_order_domain_pick' | translate }}">
+            </oui-select>
+        </oui-step-form>
+        <oui-step-form
+            data-header="{{:: 'mxPlan_order_offer' | translate }}"
+            data-on-focus="$ctrl.displaySelectedOffer = false"
+            data-on-submit="$ctrl.displaySelectedOffer = true"
+            data-editable="!$ctrl.loading.checkout">
+            <p data-translate="mxPlan_order_offer_description_domain"></p>
+            <div class="row">
+                <oui-select-picker
+                    data-ng-if="$ctrl.displaySelectedOffer"
+                    class="mb-3 col-md-4"
+                    data-label="{{ $ctrl.model.product.displayName }}"
+                    name="{{:: $ctrl.model.product.planCode }}"
+                    data-model="$ctrl.model.product"
+                    data-values="[$ctrl.model.product]"
+                    data-variant="light">
+                    <oui-select-picker-footer>
+                        <ovh-manager-order-catalog-price
+                            data-user="$ctrl.user"
+                            data-price="$ctrl.model.product.getPrice('installation')"
+                            data-tax="$ctrl.getProductTaxAmount($ctrl.model.product)">
+                        </ovh-manager-order-catalog-price>
+                    </oui-select-picker-footer>
+                </oui-select-picker>
+
+                <oui-select-picker
+                    data-ng-if="!$ctrl.displaySelectedOffer"
+                    class="mb-3 col-md-4"
+                    data-ng-repeat="product in $ctrl.products track by product.planCode"
+                    data-label="{{ product.displayName }}"
+                    name="{{:: product.planCode }}"
+                    data-model="$ctrl.model.product"
+                    data-values="[product]"
+                    data-variant="light">
+                    <oui-select-picker-footer>
+                        <ovh-manager-order-catalog-price
+                            data-user="$ctrl.user"
+                            data-price="product.getPrice('installation')"
```

Comment:
Shouldn't be:

```suggestion
                            data-price="$ctrl.model.product.getPrice('installation')"
```
?

Reply:
We're in a `ng-repeat`